pep-0581.rst

```diff
@@ -0,0 +1,310 @@
+  (few) maintainers in terms of reviewing, testing, and applying patches.
+
+- At its current state, it is not equipped to accept lots of contributions from
+  people who aren't already familiar with the code base.
+
+- The upstream roundup is in mercurial. If bpo does move to GitHub, it will become
+  difficult to update patches from upstream.
+
+- The user interface needs update and redesign. It will require UX/UI research
+  to keep it up to date with current web standards.
+
+- Email address is exposed. There is no choice to mask it.
+
+- There is no REST API available. There is an open issue in Roundup for adding
+  REST API  [#roundup-rest-api]_. Last activity was in 2016.
+
+- I personally don't want to receive email each time someone added themselves as "nosy"
+  An issue has been filed in upstream roundup since 2012 with little traction [#nosy-list]_.
+
+- Creating an account has been a hassle. We've had reports when people have
+  trouble creating accounts or logging in.
+
+Why not GitLab
+--------------
+
+Had we migrated to GitLab instead of GitHub in 2017, this PEP would have been
+titled "Using GitLab Issues for CPython".
+
+Why not other issue tracker
+---------------------------
+
+Using another issue tracker will require yet another learning curve, of having
+to learn and get used to a different interface. We'll also need to learn and
+figure out how to build the integrations with GitHub.
+
+By using GitHub issues, where the source code is currently hosted, where pull requests
+are taking place, we'll be providing consistent experience to contributors and
+maintainers, and not having to jump from one interface to another.
+
+If someone else wants to do the research of what it takes for Python to start using
+other issue tracker, and what the migration plan is, please go ahead and come
+back with a PEP.
+
+Why not focus on improving Roundup / bpo
+----------------------------------------
+
+GitHub has many features we like that are already available. We still need to
+build out additional integrations and update our bots, but this is something
+we already know how to do.
+
+In order to really improve Roundup / bpo, it needs to first migrate to GitHub,
+add CI and bots. As I understand it, there is hesitation because upstream Roundup
+is still im mercurial. Someone more familiar with Roundup / bpo needs
+to champion this effort. (I'm not volunteering, I'm sorry).
+
+I believe the effort of creating and maintaining GitHub integrations and bots
+is much less than the effort needed to get Roundup up to speed and then to continue
+maintaining it.
+
+Migration Plan
+==============
+
+Backup of GitHub data
+---------------------
+
+This effort has been started and being tracked as an issue in core-workflow [#backup-github]_.
+We're using GitHub Migrations API [#gh-migrations-api]_ to download GitHub data
+for CPython on daily basis. The archives will be dropped in an S3 bucket.
+
+Thanks to Ernest W. Durbin III for working on this.
```

Comment:
Download data: ok, but how can we make sure that data are usable? Is it possible to get a read-only review of these backups?

Reply:
See https://github.com/python/core-workflow/issues/20#issuecomment-396238565 for what kind of data are getting backed up. Perhaps ask Ernest about getting a copy of the backup if you're interested?